content/sensu-enterprise/2.8/integrations/influxdb.md

```diff
@@ -104,7 +104,7 @@ description    | Configurable custom tags (key/value pairs) to add to every Infl
 required       | false
 type           | Hash
 default        | {{< highlight shell >}}{}{{</ highlight >}}
-example        | {{< highlight json >}}
+example        | {{< highlight shell >}}
```

Comment:
Curious about this: Why change to shell?

Reply:
It conforms what I have seen in other examples within tables. A few examples before I started this effort:

https://github.com/sensu/sensu-docs/blame/master/content/sensu-core/1.4/reference/checks.md#L673-L687
https://github.com/sensu/sensu-docs/blame/master/content/sensu-core/1.4/reference/checks.md#L795-L801

I think though now that you have brought it up, it requires further discussion on when we use JSON take vs shell. IMO, it should be:

- JSON tags for full, valid JSON
- shell tags for JSON objects that can go in as Sensu attributes

If we made them all JSON tags, we'd have to add `{}` to all the JSON objects that can be Sensu attributes.

Reply:
That makes sense to me! Thanks for elaborating.